src/graphql/fragments/loanSearchCriteriaFields.graphql

```diff
@@ -0,0 +1,52 @@
+fragment loanSearchCriteriaFields on LoanSearchCriteria {
```

Comment:
What's a fragment?

Reply:
Fragments in graphql create a reusable sub-set of query fields, typically on graphql object type that is used in multiple places. The LoanSearchCriteria is a common object that can be found as a sub-field in various locations throughout the api so this fragment could be useful there. @michelleinez this may be helpful for you at some point when pulling the LSC off of a managed account.